### ISSUE TITLE
Azure Storage Explorer 1.15.1 is missing "List" permission when creating SAS on a VHD

### ISSUE THREAD

**Issue by c2company** [microsoft/AzureStorageExplorer#3747]
**Storage Explorer Version**: 1.15.1
**Build Number**:  20200902.9 
**Platform/OS**: MacOS 10.15.7
**Architecture**: x64
**Regression From**:    Not sure. Updated maybe a month ago.

## Bug Description ##
The partner portal requires r and l options when creating Shared Access Signatures when sharing images for Marketplace submission.  I now get the error:
Permissions (sp) must include read "r" and list "l".
And there is no option to select "List" in the list of permissions.

## Steps to Reproduce ##
Right click on a VHD and select create shared access signature.  Under permissions, "list" has been removed. 

## Expected Experience ##
List and Read used to be selected by default.

**Comment by JasonYeMSFT** [comment]
@c2company We are aware of this issue and is discussing with the corresponding team. List permission is a container level SAS permission so it is removed as we remove the "container-level" option. Ideally speaking you shouldn't need a container-level SAS because it will grant access to all blobs in that container, which is far beyond what is really needed for the submission. The requirement of container-level SAS was due to some legacy service issue and depending on whether they can fix it in time, we will consider adding it back.

**Comment by N3WWN** [comment]
Version 1.17.1 still has breaks the SAS URIs that are required for publishing images within Partner Portal.  

The missing requirements (which were available and working in 1.12.x) are (1) list permission and (2) the URI must be container level.

While generating the SAS URIs via Portal's web-based "Storage Explorer (Preview)" is a workaround, the Storage Explorer application should not be handicapped in this way.  I use AZE for everything... but cannot use it for generating the SAS URIs to publish images in Partner Portal since upgrading past 1.12.x.

**Comment by N3WWN** [comment]
This is still an issue in 1.18.1.

Any idea when this will get fixed?

**Comment by N3WWN** [comment]
Yup, "Storage Explorer (Preview)" is a valid workaround, but shouldn't be necessary as the functionality was part of ASE as of 1.12 and had been removed for some reason.

**Comment by N3WWN** [comment]
This is still an issue in 1.19.0.

Any idea when this will get fixed?

**Comment by JasonYeMSFT** [comment]
We are still working with the team providing the VHD submission service for a coordinated solution.

**Comment by N3WWN** [comment]
@JasonYeMSFT Any update on this?  

The Portal web-based "Storage Explorer (Preview)" is no longer working to generate Partner Center compatible SAS URIs.  

I just tried `az storage blob generate-sas ...` and that doesn't work for read+list permissions either.

To the best of my knowledge, there is no longer _**any**_ way to generate SAS URIs with read+list permissions which means publishing of new images via Microsoft Partner Portal are entirely **_broken_** at the moment.

**Comment by JasonYeMSFT** [comment]
@N3WWN According to our conversation with the service team, it should now be able to accept SAS url with only read permission right now. Is the submission process you are using still require a SAS with both read and list permission?

**Comment by N3WWN** [comment]
Hi @JasonYeMSFT !

Yes...This is the result of attempting to save when adding new images to the technical configuration of a plan:

![image](https://user-images.githubusercontent.com/19431804/151625295-e5df2566-e946-423e-a663-967a52a6feb6.png)

This error was from this afternoon.
